notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>according to the plot above <code>ttm_days[ttm_days &lt; 100].hist(bins=30)</code> about ~70% of all PR's close within the first 3 days. Days is probably a good starting point (hours might ultimately be better). But in either case, if you are going to decide on days as your unit of prediction why not plot the days distribution and make a histogram where each bucket is equal to 1 day ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Sure, we can make that plot</p>